snekbox/nsjail.py

```diff
@@ -152,8 +168,22 @@ def python3(
                 *nsjail_args,
             )
 
-        with NamedTemporaryFile() as nsj_log:
-            args = (
+        with NamedTemporaryFile() as nsj_log, MemFS(self.memfs_instance_size) as fs:
+            # Add the temp dir to be mounted as cwd
+            nsjail_args = (
+                # Mount `home` with Read/Write access
+                "--bindmount",
+                f"{fs.home}:home",
+                # Set cwd to temp dir
+                "--cwd",
+                "home",
+                # Some packages rely on the HOME env variable
+                "--env",
+                "HOME=home",
```

Comment:
Why not set these in the config file instead?

Reply:
Moved cwd and home env to config file in [c30c0eb](https://github.com/python-discord/snekbox/pull/159/commits/c30c0eb443f3ab759e9dfcbfee5bac6f4c428048)